hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/AbstractSerializationServiceTest.java

```diff
@@ -60,9 +60,12 @@
 
     @Before
     public void setup() {
+        abstractSerializationService = getAbstractSerializationService();
+    }
+
+    protected AbstractSerializationService getAbstractSerializationService() {
```

Comment:
I would not call this method 'getAbstractSerializationService', but 'buildAbstractSerializationService' since the real intent is to build something.